sdk/src/Services/SQS/Custom/Model/GetQueueAttributesResponse.extensions.cs

```diff
@@ -178,6 +172,28 @@ public string Policy
             }
         }
 
+        /// <summary>
+        /// Whether or not the queue is a First-In-First-Out (FIFO) queue
+        /// </summary>
+        public bool? FifoQueue
+        {
+            get
+            {
+                return getAttributeValue(SQSConstants.ATTRIBUTE_FIFO_QUEUE)?.ToUpper().Equals("TRUE");
+            }
+        }
+
+        /// <summary>
+        /// Whether or not the First-In-First-Out (FIFO) queue has content based deduplication enabled
```

Comment:
Ah, I understand where you are coming from. I have made the changes in c30d5bb :)